doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -986,26 +981,20 @@ Persistence of notations
 
 Notations disappear when a section is closed.
 
-.. cmd:: Local Notation @notation
-
-   Notations survive modules unless the command ``Local Notation`` is used instead
-   of :cmd:`Notation`.
-
-.. cmd:: Local Declare Custom Entry @ident
-
-   Custom entries survive modules unless the command ``Local Declare
-   Custom Entry`` is used instead of :cmd:`Declare Custom Entry`.
-
 .. _Scopes:
 
 Interpretation scopes
 ----------------------
 
+   .. todo Would "notation scope" be a better term?  Do we have multiple types of scope?
+
    .. insertprodn scope scope
 
    .. prodn::
       scope ::= @ident
 
+   .. todo explain whether scope names and scope keys are interchangeable
```

Comment:
No. For instance the following fails:

```coq
Check (1 + 1)%nat_scope.
```

It should be instead:

```coq
Check (1 + 1)%nat.
```

The following raises a warning (because it is implicitly declaring a new scope `nat`):

```coq
Notation "x @ y" := (Nat.add x y) (at level 200) : nat.
```

This should be instead:

```coq
Notation "x @ y" := (Nat.add x y) (at level 200) : nat_scope.
```

Reply:
I know the distinction is confusing...